tests/consensus/test_soft_voided4.py

```diff
@@ -12,8 +12,8 @@ class BaseSoftVoidedTestCase(SimulatorTestCase):
     seed_config = 5988775361793628169
 
     def test_soft_voided(self):
-        txA_hash = bytes.fromhex('4586c5428e8d666ea59684c1cd9286d2b9d9e89b4939207db47412eeaabc48b2')
-        txB_hash = bytes.fromhex('d19bee149abdce63bbfd523c90c3cf110563970a34100b2a62583a1eba48dfc8')
+        txA_hash = bytes.fromhex('1ae4ce163495279dafddca041b7c99abd71af55f29568746f3a20deead15f14d')
+        txB_hash = bytes.fromhex('6343d6549f6743bd1718d92919f5dabf6705762953aaec2b14e68cb048b4207a')
```

Comment:
Same here. Can't find txA and txB in the pdfs.

Reply:
labels added in https://github.com/HathorNetwork/hathor-core/pull/464/commits/f6e11d93a012840ad3b007baf5ffa2f22d14430f

Reply:
Thanks!